EddystoneBeacon.cpp

```diff
@@ -117,3 +144,35 @@ unsigned char EddystoneBeacon::compressURI(const char* uri, char *compressedUri,
 void EddystoneBeacon::loop() {
   this->poll();
 }
+/* loop() for TLM below:**/
+float temperature;
+int B=3975;                  //B value of the thermistor
+float resistance;
+unsigned long pdu_count = 0;
+void EddystoneBeacon::tempLoop(int analogPin) {
```

Comment:
Deleted `tempLoop()` and added `setTLMData(float temp,float bLevel)`

void EddystoneBeacon::setTLMData(float temp, float bLevel) {
  buffer[2] = ((float2fix(bLevel))&0x0000FF00)>>8; //LSB
  buffer[3] = (float2fix(bLevel))&0x000000FF;  //MSB
  buffer[4] = ((float2fix(temp))&0x0000FF00)>>8; //LSB
  buffer[5] = (float2fix(temp))&0x000000FF;  //MSB
   val = millis()/100;
  ulong2adv(buffer,10, val);  //UP-TIME SINCE POWER ON OR REBOOT
  ulong2adv(buffer,6, pdu_count);  //PDU COUNT

  _bleCharacteristic.setValue(buffer, len);
}